server/tools/docker-entrypoint.sh

```diff
@@ -219,6 +219,8 @@ if [ "$DB_VENDOR" != "h2" ]; then
     /bin/sh /opt/jboss/tools/databases/change-database.sh $DB_VENDOR
 fi
 
+sed -i '/^set keycloak/d' "${JBOSS_HOME}/bin/.jbossclirc"
```

Comment:
Got no clue what this is doing

Reply:
This command remove all strings starting with "set keycloak". This is done, because they gonna be unconditionally appended again after that.